Gems/SceneProcessing/Code/Source/Generation/Components/TangentGenerator/TangentGenerateComponent.cpp

```diff
@@ -66,8 +70,37 @@ namespace AZ::SceneGenerationComponents
         return nullptr;
     }
 
+    void TangentGenerateComponent::GetRegistrySettings(
+        AZ::SceneAPI::DataTypes::TangentGenerationMethod& defaultGenerationMethod, bool& debugBitangentFlip) const
+    {
+        if (auto settingsRegistry = AZ::SettingsRegistry::Get(); settingsRegistry != nullptr)
+        {
+            AZStd::string defaultTangentGenerationMethodString;
+            if (settingsRegistry->Get(defaultTangentGenerationMethodString, DefaultTangentGenerationKey))
+            {
+                if (AZ::StringFunc::Equal(defaultTangentGenerationMethodString, "MikkT"))
```

Comment:
it would be useful to add a warning if this string is something you don't specifically handle. Also you might want to do a case insensitive comparison, so someone can enter mikkt and have it work.

Reply:
StringFunc::Equal is case insensitive by default, but I will make it more explicit. I like the suggestion about the warning, I'll add that.